src/lustre/lustreAstDependencies.ml

```diff
@@ -616,8 +616,8 @@ let rec vars_with_flattened_nodes: node_summary -> LA.expr -> LA.SI.t list = fun
 
   (* Node calls *)
   | Call (_, i, es) ->
-(*     Format.eprintf "call expr: %a\n\n" (Lib.pp_print_list LA.pp_print_expr ";") es;
-    Format.eprintf "pos: %a, call: %a\n\n" Lib.pp_print_position p Format.pp_print_string i; *)
+    Format.eprintf "call expr: %a\n\n" (Lib.pp_print_list LA.pp_print_expr ";") es;
+    Format.eprintf "call: %a\n\n" HString.pp_print_hstring i;
```

Comment:
You should commented out these debug messages or print them through the Debug module.

Reply:
Whoops!